apps/research-service/src/validations/validators/spikes.validator.ts

```diff
@@ -23,32 +22,13 @@ interface SpikeDetectionResult extends ValidationResult {
 
 export class SpikesValidator implements Validator {
   async validatePolygon(polygonUuid: string): Promise<SpikeDetectionResult> {
-    if (PolygonGeometry.sequelize == null) {
-      throw new Error("PolygonGeometry model is missing sequelize connection");
-    }
-
-    const result = await PolygonGeometry.sequelize.query(
-      `
-        SELECT ST_AsGeoJSON(geom) as geo_json
-        FROM polygon_geometry 
-        WHERE uuid = :polygonUuid
-      `,
-      {
-        replacements: { polygonUuid },
-        type: QueryTypes.SELECT
-      }
-    );
-
-    if (result.length === 0) {
-      throw new Error(`Polygon with UUID ${polygonUuid} not found`);
-    }
-
-    const geoJson = JSON.parse((result[0] as { geo_json: string }).geo_json) as GeoJSONGeometry;
+    const geoJsonString = await PolygonGeometry.getGeoJSON(polygonUuid);
+    const geoJson = JSON.parse(geoJsonString) as GeoJSONGeometry;
```

Comment:
I think it would be reasonable to have the `getGeoJSON` method on the model do the parsing. I would be surprised if we ever want the raw string result from that function.